nixos/modules/system/boot/plymouth.nix

```diff
@@ -47,15 +47,15 @@ in
       };
 
       themePackages = mkOption {
-        default = [ nixosBreezePlymouth ];
+        default = [ ] ++ lib.optional (cfg.theme == "breeze") nixosBreezePlymouth;
```

Comment:
```suggestion
        default = lib.optional (cfg.theme == "breeze") nixosBreezePlymouth;
```

Reply:
We could just take this opportunity to get rid of `nixosBreezePlymouth` (I made it). It only exists so there was a default theme that actually displayed the logo and name nicely. If the bgrt theme works for this purpose we can kill it.

Reply:
@michaelpj I still think there could be users that want the breeze-plymouth theme?